includes/admin/class-wcs-admin-post-types.php

```diff
@@ -445,6 +445,23 @@ public function render_shop_subscription_columns( $column ) {
 			$the_subscription = wcs_get_subscription( $post->ID );
 		}
 
+		// If the subscription failed to load, only display the ID.
+		if ( empty( $the_subscription ) ) {
+			if ( 'order_title' === $column ) {
+				// translators: placeholder is a subscription ID.
+				echo '<strong>' . sprintf( esc_html_x( '#%s', 'hash before subscription number', 'woocommerce-subscriptions' ), esc_html( $post->ID ) ) . '</strong>';
+				?>
+				<div class="wcs-unknown-order-info-wrapper">
+					<a href="https://woocommerce.com/document/subscriptions/store-manager-guide/#section-18"><?php echo wcs_help_tip( sprintf( "This subscription couldn't be loaded from the database. %s Click to learn more.", '</br>' ) ); ?></a><?php // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
```

Comment:
**Nitpick:** Maybe add a `translators:` comment to explain that the `%s` = `<br>`.

Also, I think it should be `<br/>` or `<br>`, right now it looks like a closing tag, i.e. `</br>`

Reply:
Oh good catch! I just realised this tip and [the one](https://woocommerce.com/document/subscriptions/orders/#section-8) this fix was inspired by, weren't being translated. I've fixed that in b20e492.